index.js

```diff
@@ -4,16 +4,16 @@ const discovery = require('hyperdiscovery')
 
 module.exports = function (archiveOrKey, opts) {
   var archive = archiveOrKey.key ? archiveOrKey : createArchive(archiveOrKey)
```

Comment:
It'd be nice if you could pass in a hypercore key too, instead of just the feed.